doc/users/prev_whats_new/whats_new_3.5.0.rst

```diff
@@ -269,15 +351,115 @@ The `~mpl_toolkits.mplot3d.axes3d.Axes3D` class now has *computed_zorder*
 parameter. When set to False, Artists are drawn using their ``zorder``
 attribute.
 
+.. plot::
+
+    import matplotlib.patches as mpatches
+    from mpl_toolkits.mplot3d import art3d
+
+    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(6.4, 3),
+                                   subplot_kw=dict(projection='3d'))
+
+    ax1.set_title('computed_zorder = True (default)')
+    ax2.set_title('computed_zorder = False')
+    ax2.computed_zorder = False
+
+    corners = ((0, 0, 0), (0, 5, 0), (5, 5, 0), (5, 0, 0))
+    for ax in (ax1, ax2):
+        tri = art3d.Poly3DCollection([corners],
+                                     facecolors='white',
+                                     edgecolors='black',
+                                     zorder=1)
+        ax.add_collection3d(tri)
+        line, = ax.plot((2, 2), (2, 2), (0, 4), c='red', zorder=2,
+                        label='zorder=2')
+        points = ax.scatter((3, 3), (1, 3), (1, 3), c='red', zorder=10,
+                            label='zorder=10')
+
+        ax.set_xlim((0, 5))
+        ax.set_ylim((0, 5))
+        ax.set_zlim((0, 2.5))
+
+    plane = mpatches.Patch(facecolor='white', edgecolor='black',
+                           label='zorder=1')
+    fig.legend(handles=[plane, line, points], loc='lower center')
+
 Allow changing the vertical axis in 3d plots
 ----------------------------------------------
 
 `~mpl_toolkits.mplot3d.axes3d.Axes3D.view_init` now has the parameter
 *vertical_axis* which allows switching which axis is aligned vertically.
 
+.. plot::
+
+    x = np.array([0, 1, 2, 4])
```

Comment:
A plot with more directional structure is more instructive, e.g.
~~~
    phi = np.linspace(0, np.pi, 18)
    x, y, z = np.hstack([
        np.array([r*np.sin(phi), r*np.cos(phi), np.full_like(phi, 8-r)]) for r in range(8)])
    
    
    fig, axs = plt.subplots(1, 3, figsize=(9, 3),
                            subplot_kw=dict(projection='3d'),
                            gridspec_kw=dict(wspace=0.5))
    for vert_a, ax in zip(['z', 'y', 'x'], axs):
        pc = ax.scatter(x, y, z, c=z)
        ax.view_init(azim=30, elev=30, vertical_axis=vert_a)
        ax.set(xlabel='x', ylabel='y', zlabel='z',
               title=f'vertical_axis={vert_a!r}')
~~~
![grafik](https://user-images.githubusercontent.com/2836374/140611993-76a818dc-2b91-4a5a-b3cc-235f57f9e855.png)

Reply:
Done, but with some tweaks for the size; 9 inches is too wide for the docs page.